app/src/debug/res/layout/activity_development.xml

```diff
@@ -82,14 +82,23 @@
             android:text="Open an alert" />
 
         <Button
-            android:id="@+id/openLoggedIn"
+            android:id="@+id/openLoggedInWithWelcome"
             style="@style/HedvigButton.Purple"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="center_horizontal"
             android:layout_marginTop="@dimen/base_margin_double"
             android:text="Open an logged in with Welcome Screen" />
 
+        <Button
+            android:id="@+id/openLoggedInWithoutWelcome"
+            style="@style/HedvigButton.Purple"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:layout_marginTop="@dimen/base_margin_double"
+            android:text="Open an logged in without Welcome Screen" />
```

Comment:
Add a todo if the text key doesn't exists

Reply:
Files in the `debug`-folder are allowed to have lower standards intentionally 😉